chatterbot/utils/module_loading.py

```diff
@@ -1,4 +1,5 @@
-import importlib
+from importlib import import_module as import_string
+from inspect import isclass
```

Comment:
I don't believe so. I will make sure to remove them before this is merged. I'm currently getting better, results with these changes (basically attempting to use python modules for data instead of json files) but I'm still hitting an issue where importlib is returning a different sub-module depending on how it is called (works in tests, fails in application).